config/config.go

```diff
@@ -894,7 +896,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.ucfunnel.endpoint", "https://pbs.aralego.com/prebid")
 	v.SetDefault("adapters.unicorn.endpoint", "https://ds.uncn.jp/pb/0/bid.json")
 	v.SetDefault("adapters.unruly.endpoint", "https://targeting.unrulymedia.com/unruly_prebid_server")
-	v.SetDefault("adapters.valueimpression.endpoint", "https://rtb.valueimpression.com/endpoint")
+	v.SetDefault("adapters.valueimpression.endpoint", "http://useast.quantumdex.io/auction/pbs")
```

Comment:
Just to confirm that using `http` instead of `https` is intentional

Reply:
Our endpoint supports both http and https. but we want to use http to save time initiating secure connection